src/MusicStore/project.json

```diff
@@ -16,14 +16,14 @@
   "compile": [
     "../../shared/**/*.cs"
   ],
-  "tools": {
-    "dotnet-razor-tooling": "1.0.0-*"
-  },
   "content": [
     "Areas",
     "Views",
     "wwwroot",
-    "config.json"
+    "config.json",
+    "web.config",
+    "web.Debug.config",
```

Comment:
web.Debug.config and web.Release.config are not supported in Asp.Net Core. Remove them.